docs/content/latest/api/ysql/syntax_resources/the-sql-language/statements/savepoint_create.grammar.md

```diff
@@ -1,3 +1,3 @@
-```ebnf
```

Comment:
I tagged a few of these a while ago, to see if they improved the formatting in the docs — which they seemed to do. Did that change break something else?

Reply:
There's much more to this than you seem to understand. I send you a long email on the topic.